config/internal/ml-metadata/metadata-grpc.networkpolicy.yaml.tmpl

```diff
@@ -0,0 +1,27 @@
+kind: NetworkPolicy
+apiVersion: networking.k8s.io/v1
+metadata:
+  name: ds-pipeline-metadata-grpc-{{ .Name }}
+  namespace: {{ .Namespace }}
+spec:
+  podSelector:
+    matchLabels:
+      app: ds-pipeline-metadata-grpc-{{ .Name }}
+      component: data-science-pipelines
+  ingress:
+    - ports:
+        - protocol: TCP
+          port: 8080
+      from:
+        - podSelector:
+            matchLabels:
+              pipelines.kubeflow.org/v2_component: 'true'
+          namespaceSelector:
+            matchLabels:
+              kubernetes.io/metadata.name: {{ .Namespace }}
```

Comment:
this is not needed, by default if no namespaceSelector is provided, pods in the policy's own namespace are selected : 


> If NamespaceSelector is also set, then the NetworkPolicyPeer as a whole selects the Pods matching PodSelector in the Namespaces selected by NamespaceSelector. Otherwise it selects the Pods matching PodSelector in the policy's own Namespace. [[1]](https://docs.openshift.com/container-platform/4.16/rest_api/network_apis/networkpolicy-networking-k8s-io-v1.html#spec-egress-to-2)